Templates/BlobTrigger-Batch/metadata.json

```diff
@@ -5,7 +5,6 @@
     "language": "Batch",
     "category": [ "Experimental" ],
     "userPrompt": [
-        "connection",
-        "path"
```

Comment:
Why remove path, we can leave this in, Since it also populated in the function.json. It should appear in the portal pre-populated? 
This is more of a subjective opinion. But leaving it in would give the user an option to change container/blob.